activities/activity_nick_grokhowsky.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+"""
+Instructions
+
+- Functions intended to run for each scan
+  need to be named run_xxxxx
+
+- Do not modify the parameters of the run_xxx function
+  unless you know what you are doing
+  see optional parameters:
+  https://github.com/tangible-landscape/grass-tangible-landscape/wiki/Running-analyses-and-developing-workflows#python-workflows
+
+- All gs.run_command/read_command/write_command/parse_command
+  need to be passed env parameter (..., env=env)
+"""
+
+import grass.script as gs
+
+# Function to show significantly high and low pixels 
+def run_highestPixelValues(elev, env, **kwargs):
+	# Create a smooth surface from the elevation LiDAR points
+    gs.run_command(
+        'r.neighbors',
+         input=elev,
+         output='smooth',
+         method='average',
+         flags='c',
+	 env=env
+    )
+    # Calculate univariate statistics
+    stats = gs.parse_command('r.univar', map='smooth', env=env)
+    
+    # Parse the stats library and capture the values for mean and standard deviation
+    for value in stats:
+        if 'mean' in value:
+            mean = float(value[ -7: ])
+        if 'standard deviation' in value:
+            sd = float(value[ -7: ])
```

Comment:
Use -g flag for parsable output:
```suggestion
    stats = gs.parse_command('r.univar', map='smooth', flags='g', env=env)
    
    # Parse the stats library and capture the values for mean and standard deviation
    mean = float(stats['mean'])
    sd = float(stats['stddev'])
```

Reply:
Thanks Anna!  Initially I did use the flag -g and stats['mean'] to get the statistics, but in Grass7.9 the parse_command outputs a library with only keys and no values.  The key was something like this: "Mean:  129.321" and the value was None. Is this submission in a non-development version?

Reply:
That doesn't sound likely, please check that again. Just tested here in dev version:
```
>>> gs.parse_command('r.univar', map='DEM', flags='g')
{'n': '15425543', 'null_cells': '3030321', 'cells': '18455864', 'min': '-16.0322494506836', 'max': '65.2768096923828', 'range': '81.3090591430664', 'mean': '12.5472227763454', 'mean_of_abs': '12.6319943213651', 'stddev': '12.424136804984', 'variance': '154.359175348958', 'coeff_var': '99.0190182038256', 'sum': '193547724.467095'}
```

Reply:
You are right. I used a different flag which caused that.  Sorry about that.  -g works in both Grass 7.8 and 7.9 for me.